src/content/doc-surrealql/statements/define/api.mdx

```diff
@@ -0,0 +1,170 @@
+---
+sidebar_position: 3
+sidebar_label: DEFINE API
+title: DEFINE API statement | SurrealQL
+description: A DEFINE API statement can be used to set endpoints with custom middleware and permissions.
+---
+import Since from '@components/shared/Since.astro'
+
+<Since v="v2.2.0" />
+
+# DEFINE API statement
+
```

Comment:
```suggestion
> [!CAUTION]
> Currently, this is an experimental feature as such, it may be subject to breaking changes and may present unidentified security issues. Do not rely on this feature in production applications. To enable this, set the `SURREAL_CAPS_ALLOW_EXPERIMENTAL` [environment variable](/docs/surrealdb/cli/start/#experimental-capabilities) to `define_api`.
```